qualtran/bloqs/state_preparation/state_preparation_alias_sampling_test.py

```diff
@@ -29,6 +30,13 @@ def test_state_prep_alias_sampling_autotest(bloq_autotester):
     bloq_autotester(_state_prep_alias)
 
 
+def test_state_prep_alias_sampling_symb():
+    bloq = _state_prep_alias_symb.make()
+    L, logL, log_eps_inv = bloq.n_coeff, bloq.selection_bitsize, bloq.mu
```

Comment:
Wasn't there an open issue due to the inequality tests? Is it worth checking the symbolic costs (evaluated for specific values) match the numerical costs?

Reply:
Added a "close enough" test with explanation of where the discrepancy comes from (mainly because symbolic PrepareUniformSuperposition is currently naive and assumes worse case complexity)